backend/src/application.rs

```diff
@@ -0,0 +1,30 @@
+use crate::database::{
+    models::{Application, NewApplication, SuperUser},
+    Database,
+};
+use rocket::{
+    form::Form,
+    post,
+    serde::{json::Json, Serialize},
+};
+
+#[derive(Serialize)]
+pub enum ApplicationError {
+    UserNotFound,
+    RoleNotFound,
+    UnableToCreate,
+}
+
+#[post("/new", data = "<new_application>")]
+pub async fn create_application(
+    new_application: Form<NewApplication>,
+    _user: SuperUser,
```

Comment:
Any user should be able to create a new application right?

Reply:
makes sense, changed it!